scripts/contiv-error-report.sh

```diff
@@ -67,7 +65,7 @@ done
 pushd "$REPORT_DIR" > /dev/null
 
 # Get all nodes in the cluster and their host IP addresses
-NODES=$(cat "$NODES_FILE" | grep -v "NAME")
+NODES=$( grep < "$NODES_FILE" -v "NAME")
```

Comment:
The grep arguments should come before the redirect, e.g.:
```
NODES=$( grep -v "NAME" < "$NODES_FILE")
```
Also, grepping without a delimiter isn't really a good idea.  You might end up filtering out things you wouldn't want.  I'm not sure what the output looks like exactly, but something like this would be better:

If the whole line consists of just "NAME":
```
NODES=$( grep -vxF "NAME" < "$NODES_FILE")
```

If the line starts with "NAME" and then is followed by a space and something else:
```
NODES=$( grep -v ^$NAME " < "$NODES_FILE")
```